orderer/common/multichannel/ledger_resources.go

```diff
@@ -24,10 +24,10 @@ func checkResources(res channelconfig.Resources) error {
 		return errors.New("config does not contain orderer config")
 	}
 	if err := oc.Capabilities().Supported(); err != nil {
-		return errors.Wrapf(err, "config requires unsupported orderer capabilities: %s", err)
+		return errors.WithMessagef(err, "config requires unsupported orderer capabilities: %s", err)
```

Comment:
Do we need to both add a message to the existing error and then embed that same error?